apps/docs/content/guides/platform/backups.mdx

```diff
@@ -233,3 +233,23 @@ Common examples of check constraints that can result in such failures are:
 
 - validating against the current time, e.g. that the row being inserted references a future event
 - validating the contents of a row against the contents of another table
+
+#### Views that reference themselves
+
+Views that directly or indirectly reference themselves can cause **cyclic dependency errors** during a logical backup restore. These views are also **invalid and unusable** , any query against them will result in an error at runtime.
```

Comment:
```suggestion
Views that directly or indirectly reference themselves will cause logical restores to fail due to cyclic dependency errors. These views are also invalid and unusable in Postgres, and any query against them will result in a runtime error.
```

stylistically we don't seem to use bold text in this fashion in the rest of the doc. Not a big deal if you want to retain it though.

can -> will
logical backup restore -> logical restore (consistent with doc)